src/lang/lang.ml

```diff
@@ -501,6 +502,11 @@ let to_string_list l = List.map to_string (to_list l)
 let to_int_list l = List.map to_int (to_list l)
 let to_source_list l = List.map to_source (to_list l)
 
+let to_getter t =
+  match (demeth t).value with
+    | Fun ([], _, _, _) | FFI ([], _, _) -> fun () -> apply t []
```

Comment:
This seems arbitrary? What about functions with multiple arguments? We just return `fun () ->  fn` in this case?

Reply:
Not really arbitrary: a getter is either x or fun () -> x, we detect here when we are in the second case.